Sprint-3/todo-list/script.js

```diff
@@ -12,14 +45,92 @@ let todos = [
 
 populateTodoList(todos);
 
+function toggleComplete(index) {
+  // function to toggle complete status
+  todos[index].completed = !todos[index].completed;
+  populateTodoList(todos);
+}
+
+function deleteTodo(index) {
+  //function to delete specific todo
+  todos.splice(index, 1);
+  populateTodoList(todos);
+}
+populateTodoList(todos); 
 // This function will take the value of the input field and add it as a new todo to the bottom of the todo list. These new todos will need the completed and delete buttons adding like normal.
 function addNewTodo(event) {
   // The code below prevents the page from refreshing when we click the 'Add Todo' button.
   event.preventDefault();
   // Write your code here... and remember to reset the input field to be blank after creating a todo!
+    let input =
```

Comment:
At line 65-68, can you explain why you are looking for 3 different things for input?